medical_medicament_us/__openerp__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Medical Medicament - US Locale",
+    "summary": "Extension of medical_medicament that provides US Locale.",
+    "version": "9.0.1.0.0",
+    "category": "Medical",
+    "website": "https://laslabs.com",
+    "author": "LasLabs, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "medical_base_us",
+        "medical_manufacturer",
+        "medical_medication",
```

Comment:
@lasley do you think I should remove this dependency? From what I see, it doesn't look like medical_medication is required.